src/EnergyPlus/CTElectricGenerator.cc

```diff
@@ -602,219 +480,213 @@ namespace CTElectricGenerator {
-        Real64 AmbientDeltaT;        // (ATAIR) Difference between ambient actual and ambient design temperatures
-        Real64 DesignAirInletTemp;   // design turbine inlet temperature (C)
-        Real64 QLubeOilRec;          // recovered lube oil heat (W)
-        Real64 QExhaustRec;          // recovered exhaust heat (W)
-        Real64 LubeOilEnergyRec;     // recovered lube oil heat (J)
-        Real64 ExhaustEnergyRec;     // recovered exhaust heat (J)
-        Real64 MinHeatRecMdot;       // Heat Recovery Flow Rate if minimal heat recovery is accomplished
-        Real64 DesignMinExitGasTemp; // design engine stact saturated steam temp. (C)
-        Real64 ExhaustStackTemp;     // turbine stack temp. (C)
-        int HeatRecInNode;           // Heat Recovery Fluid Inlet Node Num
-        // notused  INTEGER :: HeatRecOutNode         !Heat Recovery Fluid Outlet Node Num
-        Real64 HeatRecInTemp;    // Heat Recovery Fluid Inlet Temperature (C)
-        Real64 HeatRecOutTemp;   // Heat Recovery Fluid Outlet Temperature (C)
-        Real64 HeatRecMdot;      // Heat Recovery Fluid Mass FlowRate (kg/s)
-        Real64 HeatRecCp;        // Specific Heat of the Heat Recovery Fluid (J/kg-K)
-        Real64 FuelHeatingValue; // Heating Value of Fuel in (kJ/kg)
-        Real64 HRecRatio;        // When Max Temp is reached the amount of recovered heat has to be reduced.
-        // and this assumption uses this ratio to accomplish this task.
-
-        //  LOAD LOCAL VARIABLES FROM DATA STRUCTURE (for code readability)
-        MinPartLoadRat = CTGenerator(GeneratorNum).MinPartLoadRat;
-        MaxPartLoadRat = CTGenerator(GeneratorNum).MaxPartLoadRat;
-        RatedPowerOutput = CTGenerator(GeneratorNum).RatedPowerOutput;
-        MaxExhaustperCTPower = CTGenerator(GeneratorNum).MaxExhaustperCTPower;
-        DesignAirInletTemp = CTGenerator(GeneratorNum).DesignAirInletTemp;
-        if (CTGenerator(GeneratorNum).HeatRecActive) {
-            HeatRecInNode = CTGenerator(GeneratorNum).HeatRecInletNodeNum;
-            HeatRecInTemp = Node(HeatRecInNode).Temp;
-
-            HeatRecCp = GetSpecificHeatGlycol(PlantLoop(CTGenerator(GeneratorNum).HRLoopNum).FluidName,
-                                              HeatRecInTemp,
-                                              PlantLoop(CTGenerator(GeneratorNum).HRLoopNum).FluidIndex,
-                                              RoutineName);
+        // min allowed operating frac full load
+        Real64 minPartLoadRat = this->MinPartLoadRat;
+
+        // max allowed operating frac full load
+        Real64 maxPartLoadRat = this->MaxPartLoadRat;
+
+        // Generator nominal capacity (W)
+        Real64 ratedPowerOutput = this->RatedPowerOutput;
+
+        // MAX EXHAUST FLOW PER W POWER OUTPUT COEFF
+        Real64 maxExhaustperCTPower = this->MaxExhaustperCTPower;
+
+        // design turbine inlet temperature (C)
+        Real64 designAirInletTemp = this->DesignAirInletTemp;
```

Comment:
Kinda seems like these locals aren't even really needed, but maybe there's some logic down below where it necessitates putting the values on locals.